sdk/cosmos/azure-cosmos-spark_3-3_2-12/README.md

```diff
@@ -0,0 +1,146 @@
+# Azure Cosmos DB OLTP Spark 3 connector
+
+## Azure Cosmos DB OLTP Spark 3 connector for Spark 3.3
+**Azure Cosmos DB OLTP Spark connector** provides Apache Spark support for Azure Cosmos DB using 
+the [SQL API][sql_api_query].
+[Azure Cosmos DB][cosmos_introduction] is a globally-distributed database service which allows 
+developers to work with data using a variety of standard APIs, such as SQL, MongoDB, Cassandra, Graph, and Table.
+
+If you have any feedback or ideas on how to improve your experience please let us know here:
+https://github.com/Azure/azure-sdk-for-java/issues/new
+
+### Documentation
+
+- [Getting started](https://aka.ms/azure-cosmos-spark-3-quickstart)
+- [Catalog API](https://aka.ms/azure-cosmos-spark-3-catalog-api)
+- [Configuration Parameter Reference](https://aka.ms/azure-cosmos-spark-3-config)
+
+[//]: # (//TODO: add more sections)
```

Comment:
Should we link instead to Spark 3.2 links? They will be mostly compatible.

Reply:
These seem to link to the base project. Not sure what your intention was here @FabianMeiswinkel?

Reply:
Never mind - I meant the empty TODO with add more sections comments - but just saw they exist in Spark 3.2 and 3.1 README as well :-). So, ignore this comment.